### PR TITLE
fix: #27917

### LINKED ISSUE #27917: Document mistakes in the Dependency injection
There is a mistake in the section [Disposal of services](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-7.0#disposal-of-services).
According to the define of Service3 , the expected output in the Index page is Service3: "Service3: IndexModel.OnGet, MyKey =" instead of "Service3: IndexModel.OnGet" which the document said.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0d78f540-81fd-1ba9-cb90-591e4ec27d9b
* Version Independent ID: 2251f557-dd3a-ec35-2558-6243337db009
* Content: [Dependency injection in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-7.0)
* Content Source: [aspnetcore/fundamentals/dependency-injection.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/dependency-injection.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
[Associated WorkItem - 88860](https://dev.azure.com/msft-skilling/Content/_workitems/edit/88860)

### PR DESCRIPTION
**Pull request by liaosunny123**
Fixes #27917 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/dependency-injection.md](https://github.com/dotnet/AspNetCore.Docs/blob/43daa0d80496870da4b665d7bbd677cb94b95b1f/aspnetcore/fundamentals/dependency-injection.md) | [Dependency injection in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?branch=pr-en-us-29182) |

### CODE REVIEW

**Comment by liaosunny123** [comment]
> 

@dotnet-policy-service agree

File: aspnetcore/fundamentals/dependency-injection.md [hunk 1]
```diff
@@ -628,7 +628,7 @@ The debug console shows the following output after each refresh of the Index pag
 ```console
 Service1: IndexModel.OnGet
 Service2: IndexModel.OnGet
-Service3: IndexModel.OnGet
+Service3: IndexModel.OnGet, MyKey = {MyKey}  //The "MyKey" is according to your config
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/dependency-injection.md:631]: Let's keep this code, run it, save the output and fix up the output.
Use 

```
{
  "MyKey": "MyKey from appsettings.Developement.json",
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "System": "Debug",
      "Microsoft": "Error"
    }
  }
}
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnetcore/fundamentals/dependency-injection.md (hunk 1 above)
Comment (liaosunny123) [reply aspnetcore/fundamentals/dependency-injection.md]: Ok, and I change it. Also I find that there is a same mistake in .net 3.x docs and fix it, too.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Thanks for the contribution. Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

Check out [is:issue is:open label:"code help wanted" "](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+label%3A%22code+help+wanted%22+) and [is:open is:issue label:"Good first issue"](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22)

**Pull request merged**